tests/sentry/eventstream/kafka/test_postprocessworker.py

```diff
@@ -32,22 +34,50 @@ def kafka_message_payload():
     ]
 
 
-@patch("sentry.eventstream.kafka.postprocessworker.dispatch_post_process_group_task")
-def test_post_process_forwarder(dispatch_post_process_group_task, kafka_message_payload):
-    """
-    Test that the post process forwarder calls dispatch_post_process_group_task with the correct arguments
-    """
-    forwarder = PostProcessForwarderWorker(concurrency=1)
+@pytest.fixture
+def kafka_message_without_transaction_header(kafka_message_payload):
+    mock_message = Mock()
```

Comment:
I tried using actual confluent_kafka.Message but it does not work. For example the set_headers expect Message objects rather than the actual headers. When I create a Message and pass it to set_headers, it barfs saying "cannot create 'cimpl.Message' instances". 

I feel its fine to use mocks here, since they return values which would have been returned by the actual message when the api's like headers(), values() are called.